community/planning/batch_processor.md

```diff
@@ -0,0 +1,84 @@
+<!--
+  Copyright 2021 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+# Batch Processor
```

Comment:
Rename to Batch Submitter (at least for now); the current component that is called Batch Submitter can be renamed. "Batch Processor" is too close to "Transaction Processor".

Reply:
renamed to batch submitter